changelogs/unreleased/gh-7319-fix-http.client-reply-proto-parser.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/lib
+
+* Fix http.client to properly parse HTTP status header like 'HTTP/2 200' where
+  version does not have minor part. (gh-7319)
```

Comment:
Please write dot in the end of statement. My fault, I missed it in your previous patch set.
Just take a look how other changelogs are formatted.